Makefile

```diff
@@ -0,0 +1,52 @@
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es3.json --env es3
```

Comment:
Do we intend to run benchmarks in the browser? If not we could consider `--env es6`.